toshling/_client.py

```diff
@@ -1,8 +1,17 @@
-from . import _endpoints as endpoints
-import requests
 import json
-from statham.schema.elements import Object
+import re
+
+import requests
 from statham.schema.constants import NotPassed
+from statham.schema.elements import Object
+from statham.schema.validation import format_checker
+
+from . import _endpoints as endpoints
+
+
+@format_checker.register("date")
+def is_date(value: str) -> bool:
+    return bool(re.match(r"\d{4}-\d{2}-\d{2}", value))
```

Comment:
Can I suggest using Python's `datetime.strptime`? That way we can validate what is passed in is an actual valid date. Using just regex means dates like 2021-02-30 will validate.

```python
try:
    datetime.strptime(value, '%Y-%m-%d')
    return True
except ValueError:
    return False
```

Reply:
Good suggestion. Do you wat me to correct in my branch and update the PR?